repair/repair.cc

```diff
@@ -938,6 +938,11 @@ static future<tasks::task_manager::task_ptr> start_repair_task(tasks::task_manag
     co_return task;
 }
 
+future<> range_repair_task_impl::run() {
```

Comment:
Please provide a more verbose git commit message to explain the motivation behind it.